app/views/questions/_comment.html.erb

```diff
@@ -1,13 +1,27 @@
 <div>
     <% @question.comments.each do |comment| %>
-    <h4>Comment by <%= comment.user.username%></h4>
+
+    <p>
+        <% if comment.user.avatar.attached? %>
+					<%= image_tag ques.user.avatar, class: "rounded-circle", height: 25 %>
```

Comment:
@andicodetrf 
encountered error when clicking on any of vote card. due to line 6

![Screenshot 2020-08-11 at 10 56 44 PM](https://user-images.githubusercontent.com/11924643/89913127-ef9a7800-dc25-11ea-83ec-e7d26b0ad6f3.png)